src/astro_image_display_api/dummy_viewer.py

```diff
@@ -123,6 +123,15 @@ def load_fits(self, file: str | os.PathLike) -> None:
             The FITS file to load. If a string, it can be a URL or a
             file path.
         """
+        if isinstance(file, (str, os.PathLike)):
```

Comment:
Do we not need to check for file extension? I guess maybe not for a dummy viewer, but probably necessary for actual implementation.

Reply:
Yeah, the dummy tries to do nothing substantive...